src/constants/tokens/swap/mainnet.ts

```diff
@@ -1,5 +1,2354 @@
+import { Token } from 'types';
+
 import { MAINNET_TOKENS } from '../common/mainnet';
-import { MAINNET_PANCAKE_SWAP_TOKENS } from './mainnetPancakeSwapTokens';
+
+const MAINNET_PANCAKE_SWAP_TOKENS = {
```

Comment:
We shouldn't need to do this, the script was failing in CI

Reply:
What was the error thrown by the process?

I'd like to keep the existing logic which generates the list using PancakeSwap's list in their repo, because this way we will always be up to date with them (the list changes frequently).

Reply:
Indeed, I did this to unblock the deployment faster, but the script should be the right thing to go for. Here's the error, we can revisit it today:

```
ts-node --esm ./scripts/generatePancakeSwapTokens.mts && yarn prettier --write ./src/constants/tokens/swap/mainnetPancakeSwapTokens.ts
[91mReferenceError: exports is not defined in ES module scope
    at file:///usr/app/scripts/generatePancakeSwapTokens.mts:28:23
    at ModuleJob.run (node:internal/modules/esm/module_job:197:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:337:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:61:12)
[0m[91merror Command failed with exit code 1.
```

Reply:
Ok I see. I'll have a look at this on my side

Reply:
I found a fix for this: https://github.com/VenusProtocol/venus-protocol-interface/pull/1046
I went with the easy route and transformed the script into a plain JavaScript module. Since it's a small script, I think we're good to go with that for now